install/kubernetes/cilium/charts/hubble-relay/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: hubble-relay
+version: 1.7.90
+appVersion: 1.7.90
+tillerVersion: ">=2.7.2"
+description: Helm chart for running Hubble Relay
+sources:
+  - https://github.com/cilium/cilium
+  - https://github.com/cilium/cilium/tree/master/pkg/hubble/relay
```

Comment:
This needs to be changed every release which is not feasible. I would remove this.

Reply:
done